extensions/interactions/MathExpressionInput/directives/oppia-interactive-math-expression-input.directive.ts

```diff
@@ -148,8 +148,18 @@ angular.module('oppia').directive('oppiaInteractiveMathExpressionInput', [
             });
           };
 
+          ctrl.getLatexErrorText = function() {
+            if (guppyInstance.latex().includes('=')) {
```

Comment:
This seems fine, but we are sure that the latex code for a valid expression won't have '=' in it as part of the math latex syntax, right?

Reply:
There is no explicit check present that restricts the creator from using '=' while creating the interaction, should I add a check there as well?